src/content/docs/apis/lookup-service-api/lookup-service-api.mdx

```diff
@@ -0,0 +1,1182 @@
+
+United States (US) endpoint:
+
+```
+https://nrql-lookup.service.newrelic.com
+```
+
+[European Union](/docs/using-new-relic/welcome-new-relic/get-started/introduction-eu-region-data-center) (EU) endpoint:
+
+```
+https://nrql-lookup.service.eu.newrelic.com
+```
+
+### Endpoints
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "150px" }}>
+        Method
+      </th>
+
+      <th>
+        Endpoint
+      </th>
+
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+       [`create`](#create-and-update)
+      </td>
+
+      <td>
+        `POST /v1/accounts/YOUR_ACCOUNT_ID/TABLE_NAME`
+      </td>
+
+      <td>
+        Upload a new table.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       [`update`](#create-and-update)
+      </td>
+
+      <td>
+        `PUT /v1/accounts/YOUR_ACCOUNT_ID/TABLE_NAME`
+      </td>
+
+      <td>
+        Replace an existing table.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       [`read`](#read)
+      </td>
+
+      <td>
+        `GET /v1/accounts/YOUR_ACCOUNT_ID/TABLE_NAME`
+      </td>
+
+      <td>
+        Download a table that was previously uploaded.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       [`delete`](#delete)
+      </td>
+
+      <td>
+        `DELETE /v1/accounts/YOUR_ACCOUNT_ID/TABLE_NAME`
+      </td>
+
+      <td>
+        Delete the given table.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       [`list`](#list)
+      </td>
+
+      <td>
+        `GET /v1/accounts/YOUR_ACCOUNT_ID`
+      </td>
+
+      <td>
+        List the tables previously updated for this account.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+The variables required in the above Lookups Service API endpoints are defined below. 
+
+<table>
+    <thead>
+        <tr>
+        <th style={{ width: "200px" }}>
+            Variable
+        </th>
+
+        <th style={{ width: "200px" }}>
+            Type
+        </th>
+
+        <th>
+            Description
+        </th>
+        </tr>
+    </thead>
+
+    <tbody>
+        <tr>
+        <td>
+            `YOUR_ACCOUNT_ID`
+        </td>
+
+        <td>
+            `number`
+        </td>
+
+        <td>
+            The account to which the table belongs
+        </td>
+        </tr>
+
+         <tr>
+        <td>
+            `TABLE_NAME`
+        </td>
+
+        <td>
+            `string`
+        </td>
+
+        <td>
+            A name for the stored table.  Table names must comply with the [custom event type standards](/docs/data-apis/custom-data/custom-events/data-requirements-limits-custom-event-data/#general):
+            * Maximum length: 255
+            * Can be a combination of alphanumeric characters, underscores, and colons.
+        </td>
+        </tr>
+    </tbody>
+</table>
+
+## Authentication
+
+Your <InlinePopover type="licenseKey" /> serves to authenticate your request to the Lookups Service API and needs to be passed as an HTTP header.
```

Comment:
I think it will be confusing to some users if we reference this as a "license key", since only a user api key will work, and actual ingest _license_ keys will not. Is there a similar "userKey" popover?

At the very least, we should change the examples to reference YOUR_USER_KEY instead of YOUR_LICENSE_KEY

Reply:
It looks like there is a userKey pop-over, so can you change these to `<InlinePopover type="userKey" />`, along with updating other "license" references?

Reply:
Updated the other "license" references and changed `<InlinePopover type="licenseKey" />` to be `<InlinePopover type="userKey" />` instead.